{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -19,7 +19,7 @@ description = "{{ cookiecutter.project_short_description }}"
 {%- endfor %}
 keywords = {{ ns.keywords_trim }}
 readme = "README.rst"
-requires-python = ">={{ cookiecutter.minimum_supported_python_version }}"
+requires-python = ">={{ cookiecutter.minimum_supported_python_version }}, <{{ '{:.2f}'.format(cookiecutter.maximum_supported_python_version | float + 0.01) }}"
```

Comment:
Tested locally and renders:

```
requires-python = ">=3.11, <3.14"
```

Cross-checked the syntax with:
https://github.com/diffpy/diffpy.srmise/blob/main/pyproject.toml